cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricClient.java

```diff
@@ -399,4 +402,35 @@ public void deployApplication(Id.Application appId, AppRequest appRequest) throw
     bodyConsumer.finished(mockResponder);
     verifyResponse(HttpResponseStatus.OK, mockResponder.getStatus(), "Failed to deploy app");
   }
+
+  public void updateApplication(ApplicationId appId, AppRequest appRequest) throws Exception {
```

Comment:
Nope. We didn't have the ability to update/delete apps in TestBase. https://issues.cask.co/browse/CDAP-4925